### PR TITLE
Refresh API docs for .NET 6 GA

### PR DESCRIPTION
**Pull request by gewarren**
This refresh includes the following key changes:

- Exclude public APIs that implement the source generator
- Add 3 assemblies that were previously omitted (System.Numerics.Vectors.dll, System.IO.Compression.Brotli.dll, System.Formats.Cbor.dll)

Replaces #7384 (we removed the System.Runtime.Experimental package for this pull request).

***
 Build.Reason:Manual by Genevieve Warren
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=264139&view=results
 source_repo.branch:net6-ga
 source_repo.url:https://apidrop.visualstudio.com/_git/binaries

### CODE REVIEW

**Comment by gewarren** [comment]
Rerunning the CI job after removing the Microsoft.JSInterop.dll assembly, per #7286.

File: xml/FrameworksIndex/windowsdesktop-6.0.xml [hunk 1]
```diff
@@ -33,8 +33,6 @@
     <Assembly Name="System.Threading.AccessControl" Version="6.0.0.0" />
     <Assembly Name="System.Windows.Controls.Ribbon" Version="6.0.0.0" />
     <Assembly Name="System.Windows.Extensions" Version="6.0.0.0" />
-    <Assembly Name="System.Windows.Forms.Analyzers.CSharp" Version="6.0.0.0" />
-    <Assembly Name="System.Windows.Forms.Analyzers" Version="6.0.0.0" />
```
Comment (RussKie) [inline xml/FrameworksIndex/windowsdesktop-6.0.xml:37]: 👍

File: xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml [hunk 2]
```diff
@@ -469,31 +417,25 @@
...
+        <typeparam name="TCollection">To be added.</typeparam>
+        <typeparam name="TElement">To be added.</typeparam>
         <param name="options">To be added.</param>
-        <param name="createObjectFunc">A <see cref="T:System.Func`1" /> to create an instance of the list when deserializing.</param>
-        <param name="elementInfo">A <see cref="T:System.Text.Json.Serialization.Metadata.JsonTypeInfo" /> instance representing the element type.</param>
-        <param name="numberHandling">The <see cref="T:System.Text.Json.Serialization.JsonNumberHandling" /> option to apply to number collection elements.</param>
-        <param name="serializeFunc">An optimized serialization implementation assuming pre-determined <see cref="T:System.Text.Json.Serialization.JsonSourceGenerationOptionsAttribute" /> defaults.</param>
-        <summary>Creates metadata for types assignable to <see cref="T:System.Collections.Generic.IEnumerable`1" />.</summary>
+        <param name="collectionInfo">To be added.</param>
+        <summary>To be added.</summary>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml:427]:
```suggestion
        <summary>Creates metadata for types assignable to <see cref="T:System.Collections.Generic.IEnumerable`1" />.</summary>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml [hunk 3]
```diff
@@ -511,30 +453,24 @@
...
-        <typeparam name="TCollection">The generic definition of the type.</typeparam>
+        <typeparam name="TCollection">To be added.</typeparam>
         <param name="options">To be added.</param>
-        <param name="createObjectFunc">A <see cref="T:System.Func`1" /> to create an instance of the list when deserializing.</param>
-        <param name="objectInfo">A <see cref="T:System.Text.Json.Serialization.Metadata.JsonTypeInfo" /> instance representing the element type.</param>
-        <param name="numberHandling">The <see cref="T:System.Text.Json.Serialization.JsonNumberHandling" /> option to apply to number collection elements.</param>
-        <param name="serializeFunc">An optimized serialization implementation assuming pre-determined <see cref="T:System.Text.Json.Serialization.JsonSourceGenerationOptionsAttribute" /> defaults.</param>
-        <summary>Creates metadata for types assignable to <see cref="T:System.Collections.IList" />.</summary>
+        <param name="collectionInfo">To be added.</param>
+        <summary>To be added.</summary>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml:462]:
```suggestion
        <summary>Creates metadata for types assignable to <see cref="T:System.Collections.IList" />.</summary>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml [hunk 4]
```diff
@@ -553,31 +489,25 @@
...
+        <typeparam name="TCollection">To be added.</typeparam>
+        <typeparam name="TElement">To be added.</typeparam>
         <param name="options">To be added.</param>
-        <param name="createObjectFunc">A <see cref="T:System.Func`1" /> to create an instance of the list when deserializing.</param>
-        <param name="elementInfo">A <see cref="T:System.Text.Json.Serialization.Metadata.JsonTypeInfo" /> instance representing the element type.</param>
-        <param name="numberHandling">The <see cref="T:System.Text.Json.Serialization.JsonNumberHandling" /> option to apply to number collection elements.</param>
-        <param name="serializeFunc">An optimized serialization implementation assuming pre-determined <see cref="T:System.Text.Json.Serialization.JsonSourceGenerationOptionsAttribute" /> defaults.</param>
-        <summary>Creates metadata for types assignable to <see cref="T:System.Collections.Generic.IList`1" />.</summary>
+        <param name="collectionInfo">To be added.</param>
+        <summary>To be added.</summary>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml:499]:
```suggestion
        <summary>Creates metadata for types assignable to <see cref="T:System.Collections.Generic.IList`1" />.</summary>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml [hunk 5]
```diff
@@ -597,36 +527,28 @@
...
-        <param name="keyInfo">A <see cref="T:System.Text.Json.Serialization.Metadata.JsonTypeInfo" /> instance representing the key type.</param>
-        <param name="valueInfo">A <see cref="T:System.Text.Json.Serialization.Metadata.JsonTypeInfo" /> instance representing the value type.</param>
-        <param name="numberHandling">The <see cref="T:System.Text.Json.Serialization.JsonNumberHandling" /> option to apply to number collection elements.</param>
-        <param name="serializeFunc">An optimized serialization implementation assuming pre-determined <see cref="T:System.Text.Json.Serialization.JsonSourceGenerationOptionsAttribute" /> defaults.</param>
-        <param name="createRangeFunc">A method to create an immutable dictionary instance.</param>
-        <summary>Creates metadata for <see cref="T:System.Collections.Immutable.ImmutableDictionary`2" /> and types assignable to <see cref="T:System.Collections.Immutable.ImmutableDictionary`2" />.</summary>
+        <param name="collectionInfo">To be added.</param>
+        <param name="createRangeFunc">To be added.</param>
+        <summary>To be added.</summary>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml:540]:
```suggestion
        <summary>Creates metadata for <see cref="T:System.Collections.Immutable.ImmutableDictionary`2" /> and types assignable to <see cref="T:System.Collections.Immutable.ImmutableDictionary`2" />.</summary>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml [hunk 6]
```diff
@@ -645,33 +567,27 @@
...
         <param name="options">To be added.</param>
-        <param name="createObjectFunc">A <see cref="T:System.Func`1" /> to create an instance of the list when deserializing.</param>
-        <param name="elementInfo">A <see cref="T:System.Text.Json.Serialization.Metadata.JsonTypeInfo" /> instance representing the element type.</param>
-        <param name="numberHandling">The <see cref="T:System.Text.Json.Serialization.JsonNumberHandling" /> option to apply to number collection elements.</param>
-        <param name="serializeFunc">An optimized serialization implementation assuming pre-determined <see cref="T:System.Text.Json.Serialization.JsonSourceGenerationOptionsAttribute" /> defaults.</param>
-        <param name="createRangeFunc">A method to create an immutable dictionary instance.</param>
-        <summary>Creates metadata for non-dictionary immutable collection types.</summary>
+        <param name="collectionInfo">To be added.</param>
+        <param name="createRangeFunc">To be added.</param>
+        <summary>To be added.</summary>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml:579]:
```suggestion
        <summary>Creates metadata for non-dictionary immutable collection types.</summary>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml [hunk 7]
```diff
@@ -691,34 +607,26 @@
...
+        <typeparam name="TValue">To be added.</typeparam>
         <param name="options">To be added.</param>
-        <param name="createObjectFunc">A <see cref="T:System.Func`1" /> to create an instance of the list when deserializing.</param>
-        <param name="keyInfo">A <see cref="T:System.Text.Json.Serialization.Metadata.JsonTypeInfo" /> instance representing the key type.</param>
-        <param name="valueInfo">A <see cref="T:System.Text.Json.Serialization.Metadata.JsonTypeInfo" /> instance representing the value type.</param>
-        <param name="numberHandling">The <see cref="T:System.Text.Json.Serialization.JsonNumberHandling" /> option to apply to number collection elements.</param>
-        <param name="serializeFunc">An optimized serialization implementation assuming pre-determined <see cref="T:System.Text.Json.Serialization.JsonSourceGenerationOptionsAttribute" /> defaults.</param>
-        <summary>Creates metadata for types assignable to <see cref="T:System.Collections.Generic.IReadOnlyDictionary`2" />.</summary>
+        <param name="collectionInfo">To be added.</param>
+        <summary>To be added.</summary>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml:618]:
```suggestion
        <summary>Creates metadata for types assignable to <see cref="T:System.Collections.Generic.IReadOnlyDictionary`2" />.</summary>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml [hunk 8]
```diff
@@ -737,31 +645,25 @@
...
+        <typeparam name="TCollection">To be added.</typeparam>
+        <typeparam name="TElement">To be added.</typeparam>
         <param name="options">To be added.</param>
-        <param name="createObjectFunc">A <see cref="T:System.Func`1" /> to create an instance of the list when deserializing.</param>
-        <param name="elementInfo">A <see cref="T:System.Text.Json.Serialization.Metadata.JsonTypeInfo" /> instance representing the element type.</param>
-        <param name="numberHandling">The <see cref="T:System.Text.Json.Serialization.JsonNumberHandling" /> option to apply to number collection elements.</param>
-        <param name="serializeFunc">An optimized serialization implementation assuming pre-determined <see cref="T:System.Text.Json.Serialization.JsonSourceGenerationOptionsAttribute" /> defaults.</param>
-        <summary>Creates metadata for types assignable to <see cref="T:System.Collections.Generic.ISet`1" />.</summary>
+        <param name="collectionInfo">To be added.</param>
+        <summary>To be added.</summary>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml:655]:
```suggestion
        <summary>Creates metadata for types assignable to <see cref="T:System.Collections.Generic.ISet`1" />.</summary>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml [hunk 9]
```diff
@@ -780,20 +682,14 @@
...
+        <typeparam name="TCollection">To be added.</typeparam>
+        <typeparam name="TElement">To be added.</typeparam>
         <param name="options">To be added.</param>
-        <param name="createObjectFunc">A <see cref="T:System.Func`1" /> to create an instance of the list when deserializing.</param>
-        <param name="elementInfo">A <see cref="T:System.Text.Json.Serialization.Metadata.JsonTypeInfo" /> instance representing the element type.</param>
-        <param name="numberHandling">The <see cref="T:System.Text.Json.Serialization.JsonNumberHandling" /> option to apply to number collection elements.</param>
-        <param name="serializeFunc">An optimized serialization implementation assuming pre-determined <see cref="T:System.Text.Json.Serialization.JsonSourceGenerationOptionsAttribute" /> defaults.</param>
-        <summary>Creates metadata for types assignable to <see cref="T:System.Collections.Generic.List`1" />.</summary>
+        <param name="collectionInfo">To be added.</param>
+        <summary>To be added.</summary>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml:692]:
```suggestion
        <summary>Creates metadata for types assignable to <see cref="T:System.Collections.Generic.List`1" />.</summary>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml [hunk 10]
```diff
@@ -851,48 +747,24 @@
...
-        <param name="getter">Provides a mechanism to get the property or field's value.</param>
-        <param name="setter">Provides a mechanism to set the property or field's value.</param>
-        <param name="ignoreCondition">Specifies a condition for the property to be ignored.</param>
-        <param name="hasJsonInclude">Whether the property was annotated with <see cref="T:System.Text.Json.Serialization.JsonIncludeAttribute" />.</param>
-        <param name="numberHandling">If the property or field is a number, specifies how it should processed when serializing and deserializing.</param>
-        <param name="propertyName">The CLR name of the property or field.</param>
-        <param name="jsonPropertyName">The name to be used when processing the property or field, specified by <see cref="T:System.Text.Json.Serialization.JsonPropertyNameAttribute" />.</param>
-        <summary>Creates metadata for a property or field.</summary>
-        <returns>A <see cref="T:System.Text.Json.Serialization.Metadata.JsonPropertyInfo" /> instance intialized with the provided metadata.</returns>
+        <typeparam name="T">To be added.</typeparam>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml:753]:
```suggestion
        <typeparam name="T">The type that the converter for the property returns or accepts when converting JSON data.</typeparam>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml [hunk 11]
```diff
@@ -851,48 +747,24 @@
...
+        <param name="options">To be added.</param>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml:754]:
```suggestion
        <param name="options">The options to initialize the metadata with.</param>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml [hunk 12]
```diff
@@ -851,48 +747,24 @@
...
+        <param name="propertyInfo">To be added.</param>
+        <summary>To be added.</summary>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml:756]:
```suggestion
        <summary>Creates metadata for a property or field.</summary>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml [hunk 13]
```diff
@@ -851,48 +747,24 @@
...
+        <returns>To be added.</returns>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml:757]:
```suggestion
        <returns>A <see cref="T:System.Text.Json.Serialization.Metadata.JsonPropertyInfo" /> instance initialized with the provided metadata.</returns>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml [hunk 14]
```diff
@@ -904,38 +776,32 @@
...
+        <typeparam name="TCollection">To be added.</typeparam>
         <param name="options">To be added.</param>
-        <param name="createObjectFunc">A <see cref="T:System.Func`1" /> to create an instance of the list when deserializing.</param>
-        <param name="elementInfo">A <see cref="T:System.Text.Json.Serialization.Metadata.JsonTypeInfo" /> instance representing the element type.</param>
-        <param name="numberHandling">The <see cref="T:System.Text.Json.Serialization.JsonNumberHandling" /> option to apply to number collection elements.</param>
-        <param name="serializeFunc">An optimized serialization implementation assuming pre-determined <see cref="T:System.Text.Json.Serialization.JsonSourceGenerationOptionsAttribute" /> defaults.</param>
-        <summary>Creates metadata for types assignable to <see cref="T:System.Collections.Generic.Queue`1" />.</summary>
+        <param name="collectionInfo">To be added.</param>
+        <param name="addFunc">To be added.</param>
+        <summary>To be added.</summary>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml:793]:
```suggestion
        <summary>Creates metadata for types assignable to <see cref="T:System.Collections.Generic.Queue`1" />.</summary>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml [hunk 15]
```diff
@@ -947,38 +813,32 @@
...
+        <typeparam name="TCollection">To be added.</typeparam>
+        <typeparam name="TElement">To be added.</typeparam>
         <param name="options">To be added.</param>
-        <param name="createObjectFunc">A <see cref="T:System.Func`1" /> to create an instance of the list when deserializing.</param>
-        <param name="elementInfo">A <see cref="T:System.Text.Json.Serialization.Metadata.JsonTypeInfo" /> instance representing the element type.</param>
-        <param name="numberHandling">The <see cref="T:System.Text.Json.Serialization.JsonNumberHandling" /> option to apply to number collection elements.</param>
-        <param name="serializeFunc">An optimized serialization implementation assuming pre-determined <see cref="T:System.Text.Json.Serialization.JsonSourceGenerationOptionsAttribute" /> defaults.</param>
-        <summary>Creates metadata for types assignable to <see cref="T:System.Collections.Generic.Stack`1" />.</summary>
+        <param name="collectionInfo">To be added.</param>
+        <summary>To be added.</summary>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml:830]:
```suggestion
        <summary>Creates metadata for types assignable to <see cref="T:System.Collections.Generic.Queue`1" />.</summary>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml [hunk 16]
```diff
@@ -996,21 +856,52 @@
...
+        <Parameter Name="collectionInfo" Type="System.Text.Json.Serialization.Metadata.JsonCollectionInfoValues&lt;TCollection&gt;" />
         <Parameter Name="addFunc" Type="System.Action&lt;TCollection,System.Object&gt;" />
       </Parameters>
       <Docs>
-        <typeparam name="TCollection">The generic definition of the type.</typeparam>
+        <typeparam name="TCollection">To be added.</typeparam>
+        <param name="options">To be added.</param>
+        <param name="collectionInfo">To be added.</param>
+        <param name="addFunc">To be added.</param>
+        <summary>To be added.</summary>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml:867]:
```suggestion
        <summary>Creates metadata for types assignable to <see cref="T:System.Collections.Generic.Stack`1" />.</summary>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml [hunk 17]
```diff
@@ -122,28 +122,24 @@
...
-        <param name="elementInfo">A <see cref="T:System.Text.Json.Serialization.Metadata.JsonTypeInfo" /> instance representing the element type.</param>
-        <param name="numberHandling">The <see cref="T:System.Text.Json.Serialization.JsonNumberHandling" /> option to apply to number collection elements.</param>
-        <param name="serializeFunc">An optimized serialization implementation assuming pre-determined <see cref="T:System.Text.Json.Serialization.JsonSourceGenerationOptionsAttribute" /> defaults.</param>
-        <summary>Creates metadata for an array.</summary>
+        <typeparam name="TElement">To be added.</typeparam>
+        <param name="options">To be added.</param>
+        <param name="collectionInfo">To be added.</param>
+        <summary>To be added.</summary>
         <returns>To be added.</returns>
         <remarks>To be added.</remarks>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml:133]: The remarks for these methods in /// source should probably be imported, as they state:

"This API is for use by the output of the System.Text.Json source generator and should not be called directly."

File: xml/System.Text.Json.Serialization.Metadata/JsonCollectionInfoValues`1.xml [hunk 18]
```diff
@@ -0,0 +1,148 @@
...
+  <TypeParameters>
+    <TypeParameter Name="TCollection" />
+  </TypeParameters>
+  <Base>
+    <BaseTypeName>System.Object</BaseTypeName>
+  </Base>
+  <Interfaces />
+  <Docs>
+    <typeparam name="TCollection">To be added.</typeparam>
+    <summary>To be added.</summary>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonCollectionInfoValues`1.xml:21]:
```suggestion
    <summary>Provides serialization metadata about a collection type.</summary>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonCollectionInfoValues`1.xml [hunk 19]
```diff
@@ -0,0 +1,148 @@
...
+    <remarks>To be added.</remarks>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonCollectionInfoValues`1.xml:22]:
```suggestion
    <remarks>This API is for use by the output of the System.Text.Json source generator and should not be called directly.</remarks>
```

Comment (gewarren) [reply xml/System.Text.Json.Serialization.Metadata/JsonCollectionInfoValues`1.xml:22]: Q: should we filter this type from the docs?

File: xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml [hunk 20]
```diff
@@ -103,12 +103,12 @@
...
-      <MemberSignature Language="C++ CLI" Value="public:&#xA;generic &lt;typename TElement&gt;&#xA; static System::Text::Json::Serialization::Metadata::JsonTypeInfo&lt;cli::array &lt;TElement&gt; ^&gt; ^ CreateArrayInfo(System::Text::Json::JsonSerializerOptions ^ options, System::Text::Json::Serialization::Metadata::JsonTypeInfo ^ elementInfo, System::Text::Json::Serialization::JsonNumberHandling numberHandling, Action&lt;System::Text::Json::Utf8JsonWriter ^, cli::array &lt;TElement&gt; ^&gt; ^ serializeFunc);" />
+      <MemberSignature Language="C#" Value="public static System.Text.Json.Serialization.Metadata.JsonTypeInfo&lt;TElement[]&gt; CreateArrayInfo&lt;TElement&gt; (System.Text.Json.JsonSerializerOptions options, System.Text.Json.Serialization.Metadata.JsonCollectionInfoValues&lt;TElement[]&gt; collectionInfo);" />
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml:106]: Should we filter JsonMetadataServices type altogether?

File: xml/System.Security.Policy/Evidence.xml [hunk 21]
```diff
@@ -209,11 +209,7 @@
           <AttributeName Language="C#">[System.Obsolete]</AttributeName>
           <AttributeName Language="F#">[&lt;System.Obsolete&gt;]</AttributeName>
         </Attribute>
-        <Attribute FrameworkAlternate="dotnet-plat-ext-6.0;net-6.0">
-          <AttributeName Language="C#">[System.Obsolete("This constructor is obsolete. Use the constructor which accepts arrays of EvidenceBase instead.")]</AttributeName>
-          <AttributeName Language="F#">[&lt;System.Obsolete("This constructor is obsolete. Use the constructor which accepts arrays of EvidenceBase instead.")&gt;]</AttributeName>
-        </Attribute>
-        <Attribute FrameworkAlternate="netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8">
+        <Attribute FrameworkAlternate="dotnet-plat-ext-6.0;net-6.0;netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8">
```
Comment (gewarren) [inline xml/System.Security.Policy/Evidence.xml:212]: I'm not sure why the specific obsoletion messages got lost here for .NET Framework monikers.

File: xml/System.Formats.Cbor/CborWriter.xml [hunk 22]
```diff
@@ -299,14 +312,10 @@
...
         <exception cref="T:System.InvalidOperationException">Writing a new value exceeds the definite length of the parent data item.
-
--or-
-
-The major type of the encoded value is not permitted in the parent data item.
-
--or-
-
-The written data is not accepted under the current conformance mode.</exception>
+            -or-
```
Comment (gewarren) [inline xml/System.Formats.Cbor/CborWriter.xml:315]: Looks like the formatting improvements weren't backported, so I think I should rerun without importing XML comments. @carlossanlop

File: xml/System.Formats.Cbor/CborReader.xml [hunk 23]
```diff
@@ -179,25 +187,13 @@
...
         <exception cref="T:System.Formats.Cbor.CborContentException">The next value has an invalid CBOR encoding.
-
--or-
-
-There was an unexpected end of CBOR encoding data.
-
--or-
-
-The next value uses a CBOR encoding that is not valid under the current conformance mode.</exception>
+            -or-
```
Comment (gewarren) [inline xml/System.Formats.Cbor/CborReader.xml:193]: Formatting improvement is being overwritten here.

File: xml/System.Formats.Cbor/CborConformanceMode.xml [hunk 24]
```diff
@@ -135,15 +124,9 @@
...
-            <item>
-              <description>Maps (major type 5) must not contain duplicate keys.</description>
-            </item>
-            <item>
-              <description>Simple values (major type 7) must be encoded as small a possible and exclude the reserved values 24-31.</description>
-            </item>
-            <item>
-              <description>UTF-8 string encodings must be valid.</description>
-            </item>
+            <item>Maps (major type 5) must not contain duplicate keys.</item>
```
Comment (gewarren) [inline xml/System.Formats.Cbor/CborConformanceMode.xml:127]: This change is not rendering correctly. It should be fixed in the source code.

image

**Review by gewarren — COMMENTED** [review]
Looks like we may want to rerun without the Cbor doc comments.

File: xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml (hunk 20 above)
Comment (gewarren) [reply xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml:106]: @layomia Should we filter JsonMetadataServices and JsonCollectionInfoValues<T> from appearing on docs?

Comment (gewarren) [reply xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml:106]: @eerhardt Should we filter JsonMetadataServices and JsonCollectionInfoValues from appearing on docs? I'm guessing we can't filter the whole System.Text.Json assembly, so I can filter at the type level if necessary.

Comment (eerhardt) [reply xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml:106]: The types are marked EditorBrowsableState.Never with a remarks section that you shouldn't call them directly:

https://github.com/dotnet/runtime/blob/0681dd60aca47a1efd41d883c42c4d2afa47168c/src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonMetadataServices.cs#L11-L13

They are real public APIs, in an assembly that users reference. So I don't think they should be filtered out. If we filtered them out, then no one will be able to see those remarks.

File: xml/System.Formats.Cbor/CborConformanceMode.xml (hunk 24 above)
Comment (gewarren) [reply xml/System.Formats.Cbor/CborConformanceMode.xml]: This looks better now - https://review.docs.microsoft.com/en-us/dotnet/api/system.formats.cbor.cborconformancemode?view=dotnet-plat-ext-5.0&branch=pr-en-us-7387.

**Comment by gewarren** [comment]
@safern @carlossanlop This is good to merge from my POV. Could one of you review it so we can get it merged?

**Comment by carlossanlop** [comment]
> Line 0, Column 0: [Warning: ECMA2Yaml_UidAssembly_NotMatched] System.Security.Policy.Evidence's moniker dotnet-plat-ext-6.0 can't match any assembly.

Do we care about this?

**Review by carlossanlop — APPROVED** [review]
LGTM, I was able to reach the docs in the preview for the 3 re-added assemblies.

Only question to answer is the one about the CI warning.

**Comment by gewarren** [comment]
> > Line 0, Column 0: [Warning: ECMA2Yaml_UidAssembly_NotMatched] System.Security.Policy.Evidence's moniker dotnet-plat-ext-6.0 can't match any assembly.
> 
> Do we care about this?

This warning is unrelated to this PR. We have a lot of these warnings in the repo. They'll be addressed by https://dev.azure.com/ceapex/Engineering/_workitems/edit/491194.

**Pull request merged**